libbeat/processors/add_process_metadata/config.go

```diff
@@ -43,6 +43,19 @@ type config struct {
 
 	// Fields is the list of fields to add to target.
 	Fields []string `config:"include_fields"`
+
+	// HostPath is the path where /proc reside
+	HostPath string `config:"host_path"`
+
+	// CgroupPrefix is the prefix where the container id is inside cgroup
+	CgroupPrefixes []string `config:"cgroup_prefixes"`
+
+	// CgroupCacheExpireTime is the length of time before cgroup cache elements expire in seconds,
+	// set to 0 to disable the cgroup cache
+	CgroupCacheExpireTime int `config:"cgroup_cache_expire_time"`
```

Comment:
this can be time.Duration, we handle input parameters, for instance, "10s" as an input is understood by our config parser